src/main/java/com/wavefront/helpers/TraceTypePatternSanitizer.java

```diff
@@ -0,0 +1,59 @@
+package com.wavefront.helpers;
+
+import com.fasterxml.jackson.databind.util.StdConverter;
+import com.wavefront.SpanSender;
+import com.wavefront.TraceTypePattern;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.logging.Logger;
+
+public class TraceTypePatternSanitizer extends StdConverter<TraceTypePattern, TraceTypePattern> {
+  private static final Logger LOGGER = Logger.getLogger(SpanSender.class.getCanonicalName());
+  private final static List<TraceTypePattern.TagVariation> DEFAULT_MANDATORY_TAGS =
+      new ArrayList<>(
+          List.of(
+              new TraceTypePattern.TagVariation("application",
+                  new ArrayList<>(Arrays.asList("Application_1", "Application_2"))),
+              new TraceTypePattern.TagVariation("service",
+                  new ArrayList<>(Arrays.asList("Service_1", "Service_2")))
+          )
+      );
+
+
+  @Override
+  public TraceTypePattern convert(TraceTypePattern value) {
+
+    // Check and fix the tags set
+    // Mandatory Tags must exist in the set and must have values
+    if (value.mandatoryTags == null) {
+      value.mandatoryTags = new ArrayList<>();
+    }
+
+    DEFAULT_MANDATORY_TAGS.forEach(tagVariation -> {
+          TraceTypePattern.TagVariation tempTagVariation =
+              value.mandatoryTags.stream().
```

Comment:
could you please merge this line with the previous one as the line limit isn't exceed.